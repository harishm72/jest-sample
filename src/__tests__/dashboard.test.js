import React from 'react';
import {shallow} from 'enzyme';
import '../setupTests'
import Dashboard from '../dashboard';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';

describe('Dashboard --- Snapshot',()=>{
  it('Rednders the Dashboard Component', () => {
      const renderedValue =  renderer.create(<Router><Dashboard/></Router>).toJSON()
      expect(renderedValue).toMatchSnapshot();
  });
});

describe("Dashboard shallow render" , () => {

  let wrapper
  beforeEach(() => {
      wrapper = shallow(<Dashboard/>);
  })
})
