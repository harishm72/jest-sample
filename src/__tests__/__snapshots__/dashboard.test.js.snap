// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard --- Snapshot Rednders the Dashboard Component 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <a
      href="/todos"
      onClick={[Function]}
    >
      Go to Todos!
    </a>
  </header>
</div>
`;
